tests/run-macros/xml-interpolation-3/XmlQuote_1.scala

```diff
@@ -12,7 +12,7 @@ object XmlQuote {
       ${XmlQuote.impl('ctx, 'args)}
   }
 
-  def impl(receiver: Expr[StringContext], args: Expr[Seq[Any]]) with QuoteContext : Expr[Xml] = {
+  def impl(receiver: Expr[StringContext], args: Expr[Seq[Any]]) (using QuoteContext): Expr[Xml] = {
```

Comment:
```suggestion
  def impl(receiver: Expr[StringContext], args: Expr[Seq[Any]])(using QuoteContext): Expr[Xml] = {
```

And other similar cases